openmdao/vectors/default_transfer.py

```diff
@@ -94,7 +94,15 @@ def _setup_transfers(group):
                     if src_indices is None:
                         pass
                     elif src_indices.ndim == 1:
-                        src_indices = convert_neg(src_indices, meta_out['global_size'])
+                        if isinstance(src_indices, tuple) or isinstance(src_indices, np.ndarray):
+                            contains_slice = [True if isinstance(i, slice) else
```

Comment:
this should just call the _is_slice function.  Also, it should only call _is_slice if src_indices is a tuple or (src_indices is an ndarray AND dtype==object).